### PR TITLE
Supermassive 0.2.0

### PR DESCRIPTION
**Pull request by freiksenet**
* Lots of fixes
* Added transform for graphql fragments
* Added tool to generate implicit resolvers

### CODE REVIEW

File: packages/supermassive/package.json [hunk 1]
```diff
@@ -2,23 +2,32 @@
...
+  "type": "module",
+  "bin": {
+    "supermassive": "./bin/supermassive.js"
+  },
+  "exports": {
+    ".": {
+      "import": "./lib/index.mjs",
+      "require": "./lib/index.js"
+    }
+  },
```
Comment (alloy) [inline packages/supermassive/package.json:15]: Interesting, TIL

File: packages/supermassive/src/benchmarks/index.ts [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+const queryAnnotationSuite = new NiceBenchmark("Query annotation");
+queryAnnotationSuite.add("supermassive", () => {
+  addTypesToRequestDocument(schema, parsedQuery);
+});
+
+async function main() {
+  await queryCompilingSuite.run();
+  await queryParsingSuite.run();
+  await queryAnnotationSuite.run();
+  await queryRunningSuite.run();
```
Comment (alloy) [inline packages/supermassive/src/benchmarks/index.ts:98]: Is there a reason to run these in serial?

**Review by alloy — APPROVED** [review]
I’ve skimmed the PR for now and am fine with you self-merging this once CI is green again.

File: packages/supermassive/src/benchmarks/index.ts (hunk 2 above)
Comment (freiksenet) [reply packages/supermassive/src/benchmarks/index.ts:98]: No reason, it outputs neater when they are in serial.

**Pull request merged**